chart/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
Please split this file into two `.helmignore` and `.gitignore`. For example `Common backup files`, `Various IDEs` and etc. should be in `.gitignore` in my opinion.